mitmproxy/contentviews/__init__.py

```diff
@@ -130,8 +130,8 @@ def get_message_content_view(
     if isinstance(message, http.Message):
         http_message = message
         if ctype := message.headers.get("content-type"):
-            if ct := http.parse_content_type(ctype):
-                content_type = f"{ct[0]}/{ct[1]}"
+            if http.parse_content_type(ctype):
+                content_type = ctype
```

Comment:
It looks like our test coverage is bad here because this is not caught, but this will break a few contentviews which currently do simple string comparisons. 

Can you pull the full content type header from `http_message.headers` in the multipart contentview instead? (http_message is passed as part of in `**metadata`). Thanks!